internal/resource/config/keypair/signing/import/key_pair_signing_import_data_source.go

```diff
@@ -55,7 +55,7 @@ type keyPairsSigningImportDataSourceModel struct {
 // GetSchema defines the schema for the datasource.
 func (r *keyPairsSigningImportDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	schemaDef := schema.Schema{
-		Description: "Describes a KeyPairsSigningImport.",
+		Description: "Describes the file for importing a key pair.",
```

Comment:
This should match our standard format, plus the description in the API doc - "Describes details of a signing key pair"